Content.Server/Atmos/EntitySystems/GasTankSystem.cs

```diff
@@ -51,8 +55,19 @@ public override void Initialize()
             SubscribeLocalEvent<GasTankComponent, GasAnalyzerScanEvent>(OnAnalyzed);
             SubscribeLocalEvent<GasTankComponent, PriceCalculationEvent>(OnGasTankPrice);
             SubscribeLocalEvent<GasTankComponent, GetVerbsEvent<AlternativeVerb>>(OnGetAlternativeVerb);
+            _cvar.OnValueChanged(CCVars.AtmosTankFragment, UpdateFragment, true);
         }
 
+        public override void Shutdown()
+        {
+            base.Shutdown();
+            _cvar.UnsubValueChanged(CCVars.AtmosTankFragment, UpdateFragment);
+        }
+
+        private void UpdateFragment(bool value)
+        {
+            _tankFragmentEnabled = value;
+        }
         private void OnGasShutdown(Entity<GasTankComponent> gasTank, ref ComponentShutdown args)
```

Comment:
```suggestion
        }
        
        private void OnGasShutdown(Entity<GasTankComponent> gasTank, ref ComponentShutdown args)
```
Always a blank line between methods.